test/testcne/vec_test.c

```diff
@@ -135,6 +135,22 @@ test3(struct test_data *t)
     return ret;
 }
 
+static void
+result_print(tst_info_t *tst, int ret)
+{
+    if (tst == NULL) {
```

Comment:
We normally do not want {} around single line statements for  if, while, for, ... 

When you get a chance it would be great to fix these.

Reply:
Sorry - this is my fault - I forgot clang-format doesn't automatically catch these and I missed this :(